frontend/src/components/survey-handler/survey-handler.styles.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const QuestionImage = styled.div<{ src: string }>`
```

Comment:
```suggestion
export const QuestionImage = styled.div<QuestionImageProps>`
```

Add interface, this can help in future when added a new atribute.